features/partitioned-cookies.yml

```diff
@@ -0,0 +1,12 @@
+name: Partitioned cookies
+description: To write...
+spec: https://wicg.github.io/cookie-store/
+compat_features:
+  - api.CookieChangeEvent.changed.partitioned_property
+  - api.CookieChangeEvent.deleted.partitioned_property
+  - api.CookieStore.delete.partitioned_option
+  - api.CookieStore.get.partitioned_return_property
+  - api.CookieStore.getAll.partitioned_return_property
+  - api.CookieStore.set.partitioned_option
+  - api.ExtendableCookieChangeEvent.changed.partitioned_property
+  - api.ExtendableCookieChangeEvent.deleted.partitioned_property
```

Comment:
This looks incomplete. CHIPS is a bigger feature than the cookie store API. This is missing at least an HTTP header, but I don't know enough about it to say what else might be missing.

Reply:
Want to come back to this PR and feature, @queengooborg?

Reply:
I added http.headers.Set-Cookie.Partitioned